moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -33,11 +33,30 @@ const getHttpMethod = knackProjectId => {
  * Function to map the signal IDs from a project object into an array and return the array length.
  * @returns integer
  */
-const countSignalsInProject = project => {
-  const signalIds = project.moped_proj_features.map(
-    feature => feature.location.properties.signal_id
+const getProjectSignals = project => {
```

Comment:
rather than counting signals, now just using this array of signals throughout. because our Knack payload can hold an empty array for the signals connection field, we don't need to bother counting them.

Reply:
Yea, seeing the bigger picture here. This makes a ton of sense.